docs/modules/ROOT/pages/serverless-cluster.adoc

```diff
@@ -83,25 +84,11 @@ Be aware that the following operations use heap memory: writing, caching, or rep
 
 Additionally, Management Center has a heap memory limit of 3.7 GB shared between all clusters in an account. Running several streaming jobs with high workloads, or one uncontrolled streaming job, can lead to the temporary loss of metrics data.
```

Comment:
@FinnLiuHZ Why are we exposing this to users? We control MC and if it cannot handle a scenario we need to treat that as a product enhancement and evolve it to be better.

Reply:
Covered in a separate ticket.